.gitignore

```diff
@@ -35,4 +37,4 @@ build/
 .vscode/
 
```

Comment:
After merging into develop branch -> merge new develop into new created branches to apply the .gitignore changes